app/src/test/java/org/simple/clinic/newentry/PatientEntryUpdateTest.kt

```diff
@@ -54,4 +58,32 @@ class PatientEntryUpdateTest {
             )
         )
   }
+
+  //Tests for validation errors in Patient Entry Screen
+  @Test
+  fun `when the user leaves full name field empty, then show error`() {
+    //given
+    val errors: List<PatientEntryValidationError> = listOf(PatientEntryValidationError.FULL_NAME_EMPTY)
+    val givenModel = defaultModel
```

Comment:
We should just rename this to `model` since `givenModel` is specific to the `UpdateSpec`.

Reply:
This applies to all the tests in this PR.

Reply:
Resolved